From 4da2a46c5fce545dc618724daa0665d43426462f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E5=BF=98=E6=80=80?= <guanwentao@uniontech.com>
Date: Mon, 23 Dec 2024 15:54:48 +0800
Subject: [PATCH 304/360] deepin: arm64: config: Enable HISI crypto acct
 drivers

Enable  HISI_SEC2 HISI_ZIP HISI_HPRE HISI_TRNG
CRYPTO_CURVE25519 selected by HISI_HPRE
---
 arch/arm64/configs/deepin_arm64_desktop_defconfig | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/deepin_arm64_desktop_defconfig b/arch/arm64/configs/deepin_arm64_desktop_defconfig
index d51c375f5955..0a883e78e87b 100644
--- a/arch/arm64/configs/deepin_arm64_desktop_defconfig
+++ b/arch/arm64/configs/deepin_arm64_desktop_defconfig
@@ -4077,7 +4077,6 @@ CONFIG_CRYPTO_PCRYPT=m
 CONFIG_CRYPTO_ECDSA=m
 CONFIG_CRYPTO_ECRDSA=m
 CONFIG_CRYPTO_SM2=y
-CONFIG_CRYPTO_CURVE25519=m
 CONFIG_CRYPTO_ANUBIS=m
 CONFIG_CRYPTO_ARIA=m
 CONFIG_CRYPTO_BLOWFISH=m
@@ -4142,6 +4141,10 @@ CONFIG_CRYPTO_CRCT10DIF_ARM64_CE=m
 CONFIG_CRYPTO_DEV_CCP=y
 CONFIG_CRYPTO_DEV_CCREE=m
 CONFIG_CRYPTO_DEV_HISI_SEC=m
+CONFIG_CRYPTO_DEV_HISI_SEC2=m
+CONFIG_CRYPTO_DEV_HISI_ZIP=m
+CONFIG_CRYPTO_DEV_HISI_HPRE=m
+CONFIG_CRYPTO_DEV_HISI_TRNG=m
 # CONFIG_CRYPTO_DEV_AMLOGIC_GXL is not set
 CONFIG_SIGNED_PE_FILE_VERIFICATION=y
 CONFIG_INDIRECT_PIO=y
-- 
2.20.1

